pinterest/teletraan
# Improve UI messaging across pages in deploy board

Why is this change necessary?
- Users struggle understanding certain parts of the Teletraan UI (see attached JIRA ticket)
- Tech debt violating DRY principle leads to inconsistencies across pages

How does this PR address the issue?
- It corrects the most pertinent visual inconsistencies
- It provides clarity on use cases like stage creation and other forms
- It adds additional context to better understand errors or requirements

What effects does this change have?
- Minor UI text changes for consistency across pages
- Changing "clone_from" field in stage creation form from text field to dropdown for better usability 
- Button link to Scaling Activities in environment landing page

Test Plan:
- Ensure modified anchor tags route correctly
- Attach screenshot of new stage creation form (with dropdown)
- Ensure scaling activities works correctly

JIRA Ticket(s): [CDP-6968]( https://jira.pinadmin.com/browse/CDP-6968)
